modules/debugger/src/render-model.js

```diff
@@ -0,0 +1,140 @@
+import {compileVertexShader, compileFragmentShader} from './compile-shader';
```

Comment:
Nit: The naming of the function is converging on `draw` rather than `render` in the remaining library.